### PR TITLE
Reacting to new Hosting API

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Reacting to https://github.com/aspnet/Hosting/pull/521

### CODE REVIEW

File: samples/LargeResponseApp/Startup.cs [hunk 1]
```diff
@@ -34,5 +35,17 @@ public void Configure(IApplicationBuilder app)
                 }
             });
         }
+
+        public static void Main(string[] args)
+        {
+            var config = WebApplicationConfiguration.GetDefault(args);
+
+            var application = new WebApplicationBuilder()
+                .UseConfiguration(config)
```
Comment (Tratcher) [inline samples/LargeResponseApp/Startup.cs]: Would it make sense to have a UseConfiguration(string[] args) overload to simplify this?

Comment (Tratcher) [inline samples/LargeResponseApp/Startup.cs]: Or just inline it:
`.UseConfiguration(WebApplicationConfiguration.GetDefault(args))`

**Comment by Tratcher** [comment]
:shipit:

**Comment by halter73** [comment]
:ship:

**Comment by JunTaoLuo** [comment]
Merged to dev in https://github.com/aspnet/KestrelHttpServer/commit/bd10d507f8d9530d601ed17edce1c6d0b0e19f15

**Pull request closed without merging**